docs/api-angular/schematics/ngrx.md

```diff
@@ -17,64 +17,88 @@ Default: `+state`
 
 Type: `string`
 
-Override the name of the folder used to contain/group the NgRx files: contains actions, effects, reducers. selectors. (e.g., +state)
+The name of the folder used to contain/group the generated NgRx files.
 
 ### facade
 
 Default: `false`
 
 Type: `boolean`
 
-Create a Facade class for the the Feature (e.g., --facade).
+Create a Facade class for the the Feature.
+
+### minimal
+
+Default: `false`
+
+Type: `boolean`
+
+Only register the root state management setup or feature state.
 
 ### module
 
 Type: `string`
 
-Path to ngModule; host directory will contain the new '+state' directory (e.g., libs/comments/src/lib/comments-state.module.ts).
+The path to NgModule where the feature state will be registered. The host directory will create/use the new state directory.
 
 ### name
 
 Type: `string`
 
-Name of the NgRx feature (e.g., Products, Users, etc.). Recommended to use plural form for name.
+Name of the NgRx feature state, such as "Products" or "Users"). Recommended to use the plural form of the name.
```

Comment:
Should the feature state name be title case or all lower case, or does it even matter?